vita3k/modules/SceDriverUser/SceAppMgrUser.cpp

```diff
@@ -172,9 +181,17 @@ EXPORT(int, sceAppMgrForceUmount) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceAppMgrGameDataMount) {
-    TRACY_FUNC(sceAppMgrGameDataMount);
-    return UNIMPLEMENTED();
+EXPORT(int, sceAppMgrGameDataMount, Ptr<const char> app_path, Ptr<const char> patch_path, Ptr<const char> rif_path, char *mount_point) {
```

Comment:
```suggestion
EXPORT(int, sceAppMgrGameDataMount, const char* app_path, const char* patch_path, const char* rif_path, char *mount_point) {
```

Reply:
is sad but no works with this, get crash

Reply:
Can you show what the crash is? there is really no reason for this to cause a crash

Reply:
yes you right, i have no understund here...

Reply:
ah no ok i remember where
![image](https://github.com/Vita3K/Vita3K/assets/5261759/f6b4637d-727e-43bf-82d0-8fb4426e6004)
i have no idea if i no using Ptr <> get that and crash

Reply:
ok sorry, stupid error caused by other things strange

Reply:
done so